src/payment/strategies/klarnav2/klarnav2-payment-strategy.spec.ts

```diff
@@ -118,6 +120,10 @@ describe('KlarnaV2PaymentStrategy', () => {
             expect(scriptLoader.load).toHaveBeenCalledTimes(1);
         });
 
+        it('loads store subscribe once', () => {
+            expect(store.subscribe).toHaveBeenCalledTimes(1);
```

Comment:
you could use `store. notifyState()` to trigger the subscribe function and make sure the widget is being re-initialized.